pkgs/development/python-modules/colanderalchemy/default.nix

```diff
@@ -7,7 +7,7 @@
 }:
 
 buildPythonPackage rec {
-  pname = "colanderclchemy";
+  pname = "ColanderAlchemy";
```

Comment:
This is not a good fix as it breaks the normalistation we do. The correct way would be to change pname in fetchPypi to the uppercase string.